packages/manager/modules/sms/src/sms/batches/create/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,90 @@
+{
+  "sms_batches_create_title": "Envoyer une campagne",
+  "sms_configure": "Configurer les différents champs",
+  "sms_batches_create_error": "Une erreur est survenue lors de la création de la campagne. {{error}}",
+  "sms_batches_create_success": "Votre campagne a été créée",
+  "sms_batches_create_name": "Nom de votre campagne",
+  "sms_batches_create_sender": "Expéditeur",
+  "sms_batches_create_sender_option": "Sélectionnez un expéditeur",
+  "sms_advise_title": "Conseil :",
+  "sms_advice_info": "Utilisez de préférence un expéditeur alphabétique pour vos envois de SMS. Vous éviterez certains filtrages d’opérateurs sur les expéditeurs numériques, notamment les numéros de portable.",
+  "sms_advice_description_1": "La personnalisation de l’expéditeur alphabétique est offerte par OVH, demandez la vôtre",
+  "sms_advice_description_2": "sur cette page",
+  "sms_advice_description_3": "(validation automatique immédiate ou manuelle sous 24/48 h).",
+  "sms_receivers": "Destinataires",
+  "sms_receivers_error_required": "Le champ \"Destinataire\" est obligatoire.",
+  "sms_receivers_error_pattern": "Le numéro du destinataire doit être au format international.",
+  "sms_message_title": "Composer un message",
+  "sms_message": "Message",
+  "sms_message_placeholder": "Votre message",
+  "sms_message_characters_left": "Caractères restants :",
+  "sms_message_equivalent": "Équivaut à :",
+  "sms_message_characters_left_warning_8bit": "Votre message contient un ou plusieurs caractères spéciaux.",
+  "sms_sms": "SMS",
+  "sms_message_alert_maxlength": "Vous avez dépassé la limite des <b>255</b> SMS.",
+  "sms_message_stop_info_start": "Le message \"<b>STOP:[numéro court]</b>\"",
+  "sms_message_stop_info_tooltip": "Exemple de numéro court : 36111",
+  "sms_message_stop_info_end": "est ajouté à la fin du SMS.",
+  "sms_message_stop_description": "Pour des raisons légales, votre SMS ne sera envoyé qu’<strong>entre 8 h et 20 h, du lundi au samedi</strong>.",
```

Comment:
Those hours are the same across all impacted subsidiaries ?

Reply:
Specifications doesn't mention any change for this. Also, it has already be done, https://github.com/ovh/manager/blob/master/packages/manager/modules/sms/src/sms/sms/compose/translations/Messages_es_ES.json#L38.